src/groups/mqb/mqbblp/mqbblp_clusterqueuehelper.h

```diff
@@ -148,12 +170,17 @@ class ClusterQueueHelper BSLS_KEYWORD_FINAL
 
         bsl::vector<PendingClose> d_pendingCloseRequests;
 
+        /// The Reopening logic blocks all new Open requests and waits for
+        /// all pending Open responses before sending Reopen request.
+        int d_numOpenRequestsInFlight;
```

Comment:
sizeof is 192 now, can be reduced to 184 if fields are reordered (keep `d_state` and `d_numOpenRequestsInFlight` together)

```
        bmqp_ctrlmsg::QueueHandleParameters d_parameters;

        bsl::vector<PendingClose> d_pendingCloseRequests;
        
        /// State of the upstream.
        Enum d_state;

        /// The Reopening logic blocks all new Open requests and waits for
        /// all pending Open responses before sending Reopen request.
        int d_numOpenRequestsInFlight;
```